doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Linux/index.md

```diff
@@ -27,7 +28,7 @@ Before building the C++ endpoint SDK, install the following components on your m
 1. Install g++, [CMake](https://cmake.org/download/), [Boost](http://www.boost.org/users/download/) and [SQLite3](https://sqlite.org/download.html):
 
    ```
-   sudo apt-get install g++ cmake libboost1.55-all-dev libsqlite3-0 libsqlite3-dev
+   sudo apt-get install g++ cmake python libboost-all-dev libsqlite3-0 libsqlite3-dev
```

Comment:
Does this change mean also that for Ubuntu 14.04 libboost-all-dev now refers to libboost1.55-all-dev (or above)?